Resources/Textures/_CM14/Objects/Clothing/Mask/coif.rsi/meta.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from CM-SS13 at https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/mask.dmi",
```

Comment:
You just need to add onto this your two new sprites and that you modified them (they're considered your modified sprites)